include/dba.php

```diff
@@ -497,6 +441,272 @@ public function any_value_fallback($sql) {
 		}
 		return $sql;
 	}
+
+	/**
+	 * @brief Executes a prepared statement
+	 *
```

Comment:
Can you please add a `@usage` documentation point to explain the implicit arguments?